### ISSUE TITLE
Issue with running  `sgl_audioset.ipynb`

### ISSUE THREAD

**Issue by anarsultani97** [topel/sglthresh#1]
Hi professor Pellegrini @topel ,

Thanks for your great work and publishing code of your work here.

I have read  your [paper ](https://arxiv.org/abs/2103.00833) which solves issue of  threshold optimization for multi-label audio tagging.

I forked this repo and tried to run `sgl_audioset.ipynb ` but in datasets folder there is not AudioSet folder to run it. I also used “PANNS: Large-scale pretrained audio neural networks for audio tagging but as you know it returns linear probability of each audio class. 

I would like to get threshold values for each of 527 audio classes  acquired with SURROGATE GRADIENT LEARNING method if possible. Or if you can update readme file to help me to run  `sgl_audioset.ipynb ` file to access those threshold values by myself. I need these threshold values to binarize audio labels' probability values and I will use it in my master thesis.

Anar Sultani
